docs: add VMware Getting Started DOC-1126 (#3173)
spectrocloud/librarium: docs/docs-content/getting-started/vmware/vmware.md

---

> sidebar_label: "Deploy a Cluster to VMware"

Reviewer: The VMware card is not present on the getting started index page :)

Author: Thanks for spotting that! Added!

title: "Deploy a Cluster to VMware"
description: "Spectro Cloud Getting Started with VMware"
hide_table_of_contents: false
sidebar_custom_props:
  icon: ""
tags: ["getting-started", "vmware"]
---

> Palette supports using [VMware](https://www.vmware.com) as a data center provider. You can deploy and manage
> [Host Clusters](../../glossary-all.md#host-cluster) on VMware virtual machines.

Reviewer: ```suggestion
Palette supports integration with [VMware](https://www.vmware.com). You can deploy and manage
[Host Clusters](../../glossary-all.md#host-cluster) on VMware.
```
I think I would follow the same pattern we used for the other providers, but its up to you :)
